.github/workflows/scorecard.yml

```diff
@@ -3,16 +3,17 @@ name: Scorecard supply-chain security
 permissions: read-all
 
 on:
-  # For Branch-Protection check. Only the default branch is supported. See
-  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
-  branch_protection_rule: # To guarantee Maintained check is occasionally updated. See
+  branch_protection_rule:
+    # To guarantee Maintained check is occasionally updated. See
+    # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
 
-  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
   schedule:
     - cron: '29 9 * * 4'
   push:
     branches: [ "main" ]
-  pull_request:
+  # Enable for testing only 
+  #pull_request:
```

Comment:
Testing only code should not be in "main" should it be? Otherwise, please document with some more words how this would be used in testing.

Reply:
I can clean up the comment.

By 'testing' I meant 'testing of the action' - the intent with scorecard is to only run it on code merged into main. However when we were getting started with it, and particularly the requirement to mitigate some of the issues,  the checks were done during a PR for expediency